llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -295,6 +295,11 @@ static opt<ErrorDetailLevel> ErrorDetails(
            clEnumValN(BothDetailsAndSummary, "full",
                       "Display each error as well as a summary. [default]")),
     cat(DwarfDumpCategory));
+static opt<std::string> AggregationJsonFile(
+    "aggregate-output-file", cl::init(""),
```

Comment:
Name improvement?: "--verify-json"?

Reply:
I went with your suggested name for llvm-gsymutil for consistency: `--json-summary-file`